app/controllers/profiles_controller.rb

```diff
@@ -1,10 +1,12 @@
-class UsersController < ApplicationController
-  load_and_authorize_resource
+class ProfilesController < ApplicationController
+  authorize_resource class: 'User'
```

Comment:
This doesn't create any instance variables because we are only using `authorize_resource` instead of `load_and_authorize` resource.  You have to manually set the `@user` variable to `current_user` in each of the methods.